quota/etcd/storage/quota_storage_test.go

```diff
@@ -397,7 +397,12 @@ func TestQuotaStorage_UpdateConfigsErrors(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		if _, err := qs.UpdateConfigs(ctx, false /* reset */, test.update); !strings.Contains(err.Error(), test.wantErr) {
+		var errStr string
+		_, err := qs.UpdateConfigs(ctx, false /* reset */, test.update)
+		if err != nil {
+			errStr = err.Error()
+		}
+		if !strings.Contains(errStr, test.wantErr) {
```

Comment:
This doesn't seem right. If `test.wantErr == ""`, the test won't fail when `err != nil`.

Reply:
Reverted.

Reply:
It doesn't look very reverted...

Reply:
Ah that one was part of a different commit. I'll try again.